pyproject.toml

```diff
@@ -18,11 +18,10 @@
 #
 # Release a new version:
 #
-#  - git tag VERSION
-#  - git push --tags
-#  - Remove untracked files/dirs: git clean -fdx
-#  - python -m build
-#  - twine upload dist/*
+#  - go to the GitHub release tab: https://github.com/python/pyperformance/releases
+#  - click "Draft a new release" and fill the contents
```

Comment:
Do you mean "Create a new release"?

The URL is https://github.com/python/pyperformance/releases/new, right?

Reply:
Regarding "fill the contents", presumably you mean the following:

1. select from "Choose a tag" dropdown (or "create a new tag" via that dropdown)
2. select "Target: main" from the other dropdown (the default?)
3. enter the version number for "Release title", matching the name of the tag from (1)
4. enter something in "Describe this release"

Is the above correct?

Regarding (1), what's the best way to determine what the next version should be?  It might be obvious to many, but it would help some if this were spelled out.  Also, the tag should not exist yet right?

Regarding (4), what should go here?  It seems like, at most, it would include either a basic "what's new" list or a link to a changelog.  Leaving it blank would probably be fine too.  Is there a way to have a template for this field (and maybe for (3) the "Release title" field?

Anyway, what you have is probably fine, but it may be worth putting a bit more info here in this comment in pyproject.toml.

Reply:
Also, am I correct in thinking that the "Prepare a release" steps remain unchanged?

If we can have a "publish" template, it would be nice if also reminded the person to make sure they completed those steps already.

Reply:
For (1), you can either pick an existing tag, or create a new one - the drop down box has "Find or create a new tag". Because we've not created one yet with these steps, we want to create a new one?

For (2) and (3), sounds good.

For (4), one option is to click the "Generate release notes" button and it'll fill something in based on the PRs that were merged.